.rubocop_todo.yml

```diff
@@ -577,3 +592,8 @@ Style/SlicingWithRange:
     - 'app/models/name_parse.rb'
     - 'test/models/comment_test.rb'
     - 'test/models/image_script_test.rb'
+
+# Offense count: 218
+# Cop supports --auto-correct.
+Style/StringConcatenation:
+  Enabled: false
```

Comment:
Too many to deal with in this PR. 
I created a Pivotal Story. https://www.pivotaltracker.com/story/show/174567631